solidity/dashboard/src/services/tbtc-authorization.service.js

```diff
@@ -415,11 +415,39 @@ const fetchAvailableAmount = async (web3Context, operator) => {
   )
 }
 
+const deauthorizeSortitionPoolContract = async (
+  web3Context,
+  data,
+  onTransactionHashCallback
+) => {
+  const { keepBondingContract, yourAddress } = web3Context
+  const { operatorAddress, poolAddress } = data
+
+  await keepBondingContract.methods
+    .deauthorizeSortitionPoolContract(operatorAddress, poolAddress)
+    .send({ from: yourAddress })
+    .on("transactionHash", onTransactionHashCallback)
+}
+
+const deauthorizeTBTCSystem = async (
```

Comment:
what do you think about leaving just:
```
const deauthorizeTBTCSystem = async (
  web3Context,
  operatorAddress,
  onTransactionHashCallback
) => {
  const { keepBondingContract, yourAddress } = web3Context
  const poolAddress = await fetchSortitionPoolForTbtc(web3Context)

  await keepBondingContract.methods
    .deauthorizeSortitionPoolContract(operatorAddress, poolAddress)
    .send({ from: yourAddress })
    .on("transactionHash", onTransactionHashCallback)
}
```

Reply:
I think we can leave `deauthorizeSortitionPoolContract` function for future.

Reply:
do we already have a case in mind for `deauthorizeSortitionPoolContract` function? If yes, then lets leave it, if no, YAGNI principle says to remove it :)